bin/GameIndex.dbf

```diff
@@ -483,8 +483,8 @@ Name   = Gran Turismo 4 - Prologue
 Region = NTSC-Unk
 Compat = 5
 //eeClampMode = 3 // Text in races works
-vuClampMode = 2 // Text in GT mode works
-MemCardFilter = SCAJ-20066/SCAJ-30006/SCAJ-30007/SCAJ-30008/SCPS-15055/SCPS-17001/SCPS-19252/SCPS-19304/SCPS-15009/SCPS-55007
+//vuClampMode = 2 // Text in GT mode works
```

Comment:
What is the reason for commenting out vuclampmode?

Reply:
GT tested without vuClamp and it worked fine.

Reply:
Ditto.
Here is the AppVeyor build that has commented out memcard filter. To be tested in the future
https://ci.appveyor.com/project/gregory38/pcsx2/builds/33134819/job/bt5sgm2snxx1nasx/artifacts